kidsbook/post/views.py

```
@@ -157,10 +170,12 @@ class CommentLike(generics.ListCreateAPIView):
 
     def list(self, request, **kwargs):
         try:
-            queryset = self.get_queryset().filter(comment = Comment.objects.get(id=kwargs['pk'])).filter(like_or_dislike=True)
-        except Exception as e:
-            return Response({'error': str(e)}, status=status.HTTP_400_BAD_REQUEST)
-        serializer = CommentLikeSerializer(queryset, many=True)
+            queryset = self.get_queryset().filter(comment = Comment.objects.get(id=kwargs['pk']))
+        except Exception:
```

Can you return the exception message in the response for all `except Exception:` ?
` except Excetion as exc: ...`